capture/genBitmaps.js

```diff
@@ -224,8 +225,9 @@ function processScenario (casper, scenario, scenarioOrVariantLabel, scenarioLabe
           .replace(/\{viewportLabel\}/, makeSafe(vp.name))
           .replace(/[^a-z0-9_\-]/gi, ''); // remove anything that's not a letter or a number or dash or underscore.
 
-        if (!/\.png$/i.test(fileName)) {
-          fileName = fileName + '.png';
+        var extRegExp = new RegExp(outputFormat + '$', 'i');
```

Comment:
maybe this one liner is simpler/more readable?
`filename = filename.replace(outputFormat, '') + outputFormat;`